features/bevragen/fields-fout-cases-gba.feature

```diff
@@ -324,24 +324,375 @@ Rule: De fields parameter bevat verkorte/volledig veld paden die verwijzen naar
     | fields | fields[0] | Parameter bevat een niet toegestane veldnaam. |
 
     Voorbeelden:
-    | fields                                                 |
-    | geheimhoudingPersoonsgegevens                          |
-    | opschortingBijhouding                                  |
-    | opschortingBijhouding.reden                            |
-    | opschortingBijhouding.reden.omschrijving               |
-    | opschortingBijhouding.datum                            |
-    | opschortingBijhouding.datum.type                       |
-    | rni                                                    |
-    | rni.deelnemer                                          |
-    | rni.deelnemer.code                                     |
-    | rni.omschrijvingVerdrag                                |
-    | rni.categorie                                          |
-    | verificatie                                            |
-    | verificatie.datum                                      |
-    | verificatie.datum.datum                                |
-    | verificatie.omschrijving                               |
-    | inOnderzoek                                            |
-    | inOnderzoek.burgerservicenummer                        |
-    | inOnderzoek.datumIngangOnderzoekPersoon                |
-    | naam.inOnderzoek.voornamen                             |
-    | geboorte.inOnderzoek.datumIngangOnderzoek              |
\ No newline at end of file
+    | fields                                                                               |
+    | geheimhoudingPersoonsgegevens                                                        |
```

Comment:
Hier staan ook fields, zoals immigratie die niet voorkomen in persoon beperkt. Ik weet niet hoe de validatie in de gba api gebeurt, maar bij de proxy wordt eerst gekeken of een veld een bestaand veld is en daarna of het een toegestande veld is

Reply:
de voorbeeldenlijst stond bij het verkeerde scenario (hoort bij raadplegen). nu juiste voorbeelden bij zoeken gezet